calico_kubernetes/calico_kubernetes.py

```diff
@@ -58,7 +67,41 @@ def delete(self, pod_name, docker_id):
 
         # Remove the profile for the workload.
         self.calicoctl('container', 'remove', self.docker_id)
-        self.calicoctl('profile', 'remove', self.pod_name)
+
+        # Delete profile
+        try:
+            self._datastore_client.remove_profile(self.pod_name)
+        except:
+            print "Profile with name %s cannot be found" % self.pod_name
+
+    def _configure_profile(self, endpoint):
+        """
+        Configure the calico profile for a pod.
+
+        Currently assumes one pod with each name.
+        """
+        print('Configuring Pod Profile')
+        profile_name = self.pod_name
+
+        if self._datastore_client.profile_exists(profile_name):
+            print "A profile with name %s already exists" % profile_name
```

Comment:
Anywhere that you sys.exit, I think that needs to be clear in the logs.

So, instead of "A profile with name already exists", maybe "ERROR: A profile with name already exists, exiting."  Or something similar.